packages/local-win/season_tracker.py

```
@@ -467,13 +623,21 @@ def main_loop(cfg: Config) -> None:
             try:
                 frame = _grab_points_region(sct, cfg.monitor_index)
                 results = reader.readtext(frame)
-                wtp = parse_wtp_from_ocr(results)
+                wtp, verdict = analyze_ocr(results)
             except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)